comp/core/secrets/secretsimpl/secrets.go

```diff
@@ -366,20 +366,47 @@ func (r *secretResolver) Resolve(data []byte, origin string) ([]byte, error) {
 	return finalConfig, nil
 }
 
-// allowlistPaths restricts what config settings may be updated
-// tests can override this to exercise functionality: by setting this to nil, allow all settings
+// allowlistPaths restricts what config settings may be updated. Any secrets linked to a settings containing any of the
+// following strings will be refreshed.
+//
+// For example, allowing "additional_endpoints" will trigger notifications for:
+//   - "additional_endpoints"
+//   - "logs_config.additional_endpoints"
+//   - "logs_config.additional_endpoints.url"
+//   - ...
+//
 // NOTE: Related feature to `authorizedConfigPathsCore` in `comp/api/api/apiimpl/internal/config/endpoint.go`
```

Comment:
Could you update this comment so that it points at `comp/api/api/def/component.go` instead?